File: football-ticket-service/src/main/java/com/app/footballticketservice/dto/TeamDetailDTO.java
package com.app.footballticketservice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TeamDetailDTO {
    private String name;
    private String logo;
    private String address;
    private String goals;
    private String win;
    private String draw;
    private String losses;
    private String matchPlayed;
    private String establishedYear;
}

File: football-ticket-ui/src/app/share/model/TeamDetailDTO.ts
export class TeamDetailDTO {
  constructor(
    public lastResult: LastResultDTO = new LastResultDTO(),
    public teamDetail: TeamDetail = new TeamDetail(),
    public last5Match: LastResultDTO[] = [],
  ) {
  }
}

export class LastResultDTO {
  constructor(
    public homeTeam: string = '',
    public awayTeam: string = '',
    public homeScore: string = '',
    public awayScore: string = '',
    public matchDatetime: string = '',
    public leagueName: string = '',
    public homeLogo: string = '',
    public awayLogo: string = '',
    public status: string = '',
  ) {
  }
}

export class TeamDetail {
  constructor(
    public name: string = '',
    public logo: string = '',
    public address: string = '',
    public goals: string = '',
    public win: string = '',
    public draw: string = '',
    public losses: string = '',
    public matchPlayed: string = '',
    public establishedYear: string = '',
  ) {
  }
}
